content/first-notebook.md

```diff
@@ -7,8 +7,8 @@
 ```
 
 ```{instructor-note}
-- 10 min teaching
-- 10 min exercises
+- 5 min teaching
+- 20 min exercises
```

Comment:
```suggestion
- 15 min exercises
```

Reply:
unless we add the git init stuff there

Reply:
15 minutes because it is too little content?

Reply:
20 mins is good to get started, find the conda environment, and also to give chance to ask questions and discuss. I would not shorten that part.